src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -203,9 +209,7 @@ private SwiftCompileStep makeCompileStep(SourcePathResolver resolver) {
         });
 
     compilerCommand.addAll(Arg.stringify(compilerFlags, resolver));
-    for (SourcePath sourcePath : srcs) {
-      compilerCommand.add(resolver.getRelativePath(sourcePath).toString());
-    }
+    compilerCommand.add("-filelist", swiftFileListPath.toString());
```

Comment:
@robbertvanginkel Can we make this a config option? I think it will be useful for debugging and logging.